<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>untitled</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Search Location</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditLocation"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxExpressions">
      <property name="title">
       <string>Search Expression</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="labelSearch">
         <property name="text">
          <string>Search for:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditReplace"/>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditSearch"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelReplace">
         <property name="text">
          <string>Replace with:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="radioButtonPlain">
         <property name="text">
          <string>Plain</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="radioButtonRegex">
         <property name="text">
          <string>Regex</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="checkBoxCaseSensitive">
         <property name="text">
          <string>Case Sensitive</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxLimits">
      <property name="title">
       <string>Search limit</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelFileSize">
         <property name="text">
          <string>File size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxFileSize">
         <item>
          <property name="text">
           <string>Irrelevant</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Less than</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>More than</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Exactly</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBoxFileSize">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> KB</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLastModified">
         <property name="text">
          <string>File last modified</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxLastModified">
         <item>
          <property name="text">
           <string>Irrelevant</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Before</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>After</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>At</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDateTimeEdit" name="dateTimeEditLastModified">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxResults">
      <property name="title">
       <string>Search Results</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableViewResults"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>507</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonReplace">
      <property name="text">
       <string>Replace</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButtonSearch">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicenses"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLicenses">
   <property name="text">
    <string>Licenses</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
